packages/web3-errors/src/error_codes.ts

```diff
@@ -74,6 +74,7 @@ export const ERR_TX_DATA_AND_INPUT = 425;
 export const ERR_TX_POLLING_TIMEOUT = 426;
 export const ERR_TX_RECEIPT_MISSING_OR_BLOCKHASH_NULL = 427;
 export const ERR_TX_RECEIPT_MISSING_BLOCK_NUMBER = 428;
+export const ERR_TX_INVALID_PROPERTIES_FOR_TYPE = 429;
```

Comment:
Is it ok that we have the same code for two different errors?
`ERR_TX_INVALID_PROPERTIES_FOR_TYPE` & `ERR_TX_LOCAL_WALLET_NOT_AVAILABLE`

Reply:
No, good catch, updated [here](https://github.com/web3/web3.js/pull/6102/commits/db3f3ee30c45025364d1f7106d2ae420ec8ec401)